falcon/app.py

```diff
@@ -886,3 +886,22 @@ def _get_body(self, resp, wsgi_file_wrapper=None):
             return iterable, None
 
         return [], 0
+
+
+# NOTE(mikeyusko): This class is a compatibility alias, and it may be removed
+# in a future release.
+class API(App):
+    """
+    This class is a compatibility alias of :class:`falcon.App`.
+
+    ``API`` was renamed to :class:`App <falcon.App>` in Falcon 3.0 in order to
```

Comment:
I took the liberty to make some minor edits to this paragraph.

@kgriffs check if this looks acceptable to you.